File: src/Suframa.Sciex.UserInterface/www/app/view-model/DadosCadastroVM.ts
import { identificacaoPessoaFisicaVM } from './IdentificacaoPessoaFisicaVM';
import { identificacaoPessoaJuridicaVM } from './IdentificacaoPessoaJuridicaVM';
import { atividadesPessoaJuridicaVM } from './AtividadesPessoaJuridicaVM';
import { quadrosAdministradoresVM } from './QuadrosAdministradoresVM';
import { quadrosSocietariosVM } from './QuadrosSocietariosVM';
import { quadroPessoalVM } from './QuadroPessoalVM';
import { dadosSolicitanteVM } from './DadosSolicitanteVM';
import { documentosComprobatoriosVM } from './DocumentosComprobatoriosVM';
import { resumoVM } from './ResumoVM';
import { campoSistemaVM } from './CampoSistemaVM';
import { pendenciaCadastralVM } from './PendenciaCadastralVM';
import { protocoloVM } from './ProtocoloVM';
import { pedidoCorrecaoVM } from './PedidoCorrecaoVM';

export class dadosCadastroVM {
	atividadesPessoaJuridica: atividadesPessoaJuridicaVM;
	camposSistema: campoSistemaVM[];
	dadosSolicitante: dadosSolicitanteVM;
	documentosComprobatorios: documentosComprobatoriosVM;
	identificacaoPessoaFisica: identificacaoPessoaFisicaVM;
	identificacaoPessoaJuridica: identificacaoPessoaJuridicaVM;
	idProtocolo?: number;
	isCredenciamento: boolean;
	isInscricaoCadastral: boolean;
	isInscricaoCadastralPessoaJuridica: boolean;
	pendenciasCadastrais: pendenciaCadastralVM[];
	pedidoCorrecao: pedidoCorrecaoVM[];
	protocolo: protocoloVM;
	quadroPessoal: quadroPessoalVM[];
	quadrosAdministradores: quadrosAdministradoresVM;
	quadrosSocietarios: quadrosSocietariosVM;
	resumo: resumoVM;
	tipoDadoCadastro: number;
}

File: src/Suframa.Sciex.CrossCutting.DataTransferObject/ViewModel/DadosCadastroVM.cs
using Suframa.Sciex.CrossCutting.DataTransferObject.Enum;
using System.Collections.Generic;

namespace Suframa.Sciex.CrossCutting.DataTransferObject.ViewModel
{
	public class DadosCadastroVM
	{
		public AtividadesPessoaJuridicaVM AtividadesPessoaJuridica { get; set; }
		public IEnumerable<CampoSistemaVM> CamposSistema { get; set; }
		public DadosSolicitanteVM DadosSolicitante { get; set; }
		public DocumentosComprobatoriosVM DocumentosComprobatorios { get; set; }
		public IdentificacaoPessoaFisicaVM IdentificacaoPessoaFisica { get; set; }
		public IdentificacaoPessoaJuridicaVM IdentificacaoPessoaJuridica { get; set; }
		public int? IdProtocolo { get; set; }
		public int? IdRequerimento { get; set; }
		public EnumTipoRequerimento? IdTipoRequerimento { get; set; }
		public bool IsCredenciamento { get; set; }
		public bool IsInscricaoCadastral { get; set; }
		public bool IsInscricaoCadastralPessoaJuridica { get; set; }
		public PedidoCorrecaoVM[] PedidoCorrecao { get; set; }
		public IEnumerable<PendenciaCadastralVM> PendenciasCadastrais { get; set; }
		public ProtocoloVM Protocolo { get; set; }
		public IEnumerable<QuadroPessoalVM> QuadroPessoal { get; set; }
		public QuadrosAdministradoresVM QuadrosAdministradores { get; set; }
		public QuadrosSocietariosVM QuadrosSocietarios { get; set; }
		public IEnumerable<ResumoVM> Resumo { get; set; }
		public EnumTipoDadoCadastro TipoDadoCadastro { get; set; }
	}
}